### PR TITLE
More verbose Slack messages

### PR DESCRIPTION
**Pull request by RemiWoler**
- Added more information about the push to the event queue
- Expose that extra information in the build&suite
- Update Slack messages to include the extra information

### CODE REVIEW

**Comment by rgeraads** [comment]
Aside from code styling I would very much welcome this PR :+1:

File: aux/callback.php [hunk 1]
```diff
@@ -69,6 +69,9 @@
 }
 
 $branch = $payload['head_commit']['id'];
+$comment = $payload['head_commit']['message'];
+$author = $payload['head_commit']['author']['name'];
+$repo = $payload['repository']['name'];
```
Comment (jaspernbrouwer) [inline aux/callback.php]: Please align these.

File: aux/callback.php [hunk 2]
```diff
@@ -81,7 +84,7 @@
 
 $data = json_encode(array(
     'suite'  => $suite,
-    'params' => array('branch' => $branch)
+    'params' => array('branch' => $branch, 'comment'=>$comment, 'author'=>$author, 'repo'=>$repo)
```
Comment (jaspernbrouwer) [inline aux/callback.php]: Please add spaces around the arrows (`=>`).

File: src/F500/CI/Build/StandardBuild.php [hunk 3]
```diff
@@ -108,4 +108,19 @@ public function toJson()
...
+
+    public function getSuiteAuthor()
+    {
+        return $this->suite->getAuthor();
+    }
+
+    public function getSuiteRepo()
+    {
+        return $this->suite->getRepo();
+    }
```
Comment (jaspernbrouwer) [inline src/F500/CI/Build/StandardBuild.php:125]: Maybe add these methods to the interface as well?

File: src/F500/CI/Suite/StandardSuite.php [hunk 4]
```diff
@@ -149,4 +149,28 @@ public function toJson()
...
+        return $this->config['author'];
+    }
+
+    public function getRepo()
+    {
+        if (!isset($this->config['repo'])) {
+            return '';
+        }
+        return $this->config['repo'];
+    }
```
Comment (jaspernbrouwer) [inline src/F500/CI/Suite/StandardSuite.php:175]: Why are these methods added? I don't see them used.

**Comment by jaspernbrouwer** [comment]
Can you also add the commit hash? That's what's passed into "branch" from the payload:

``` php
$branch = $payload['head_commit']['id'];
```

If you can add this to the build as well and have a message like `%s pushed commit %s to %s`, that would be awesome!

**Pull request merged**